### ISSUE TITLE
(🐞) The Tidelift logo in the readme is stretched

### ISSUE THREAD

**Issue by KotlinIsland** [PyCQA/pylint#6064]
### Bug description

Expected:
![image](https://user-images.githubusercontent.com/65446343/160945827-7dd115d5-631f-46e4-bfb0-da835bdb1dd8.png)

Actual
![image](https://user-images.githubusercontent.com/65446343/160945805-efca2b2b-48b6-47b7-ac77-f1896196b68c.png)

### Command used

```shell
N/A
```

### Pylint output

```shell
N/A
```

### Expected behavior

N/A

### Pylint version

```shell
N/A
```
